_includes/head.html

```diff
@@ -1,4 +1,7 @@
 <head>
+  <meta name="application-name" content="Ableton on GitHub">
+  <meta name="author" content="Ableton AG">
```

Comment:
The `author` meta tag doesn't seem relevant/necessary for our site. Generally it's used for things like the author of an article, so I think this can be removed. Do you have insight on this tag's impact on SEO?